admin_ui_support/modules/admin_ui_widget_example/admin_ui_widget_example.admin_ui.components.yml

```diff
@@ -0,0 +1,4 @@
+widgets:
```

Comment:
Where should we document the new API?

Reply:
Do you agree  with a storybook being the right place?

Reply:
I think that's a good idea at least for now. It can be converted to somewhere else later if it doesn't work for us.